src/main/java/com/Feature1/WSFeaure1/WsFeaure1Application.java

```diff
@@ -0,0 +1,13 @@
+package com.Feature1.WSFeaure1;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class WsFeaure1Application {
```

Comment:
@Xeniia-droid It is better not to use `1` in the names as it can be confused with `l` easily. 

Let's name it `TestAssignmentApplication`or something.